core/planner.c

```diff
@@ -623,6 +623,9 @@ bool enough_gas(int current_cylinder)
 
```

Comment:
a429ea98
i just asked @sfuchs79 to amend his commit messages where the commit title should be a separate sentence and the commit message body should start on a new sentence. @dirkhh what do you think about commit titles warping into the commit messages like this?

just a nitpick, ultimately.